pkg/controller/workload/workload_hash.go

```diff
@@ -90,7 +90,7 @@ func (h *HashName) flushDelta(str string, num uint32) error {
 	return err
 }
 
-func (h *HashName) StrToNum(str string) uint32 {
+func (h *HashName) Hash(str string) uint32 {
```

Comment:
The function name seems a bit abstract and easy to confuse.

Reply:
on the contrary, i think this is consistent with go hash  pkg